Formula/axel.rb

```diff
@@ -12,13 +12,15 @@ class Axel < Formula
     sha256 "b0a7bcbab57df1b9877ac3a6a8343a3a5b4256b4cfa5a6bfe777b9ba92930d31" => :mavericks
   end
 
-  def install
-    system "./configure", "--prefix=#{prefix}",
-                          "--etcdir=#{etc}",
-                          "--debug=0",
-                          "--i18n=0"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext"
+  depends_on "openssl" => :recommended
+  depends_on "libressl" => :optional
```

Comment:
Unless there proves a demand for us to do so, I'd rather not introduce an optional `libressl` dependency here & simply make the `openssl` one mandatory. We're trying to step back on the LibreSSL/OpenSSL conflict situation a little until Homebrew can handle conflicting dependencies more sanely.